exercises/practice/two-fer/.docs/instructions.md

```diff
@@ -24,3 +24,5 @@ Here are some examples:
 |Bob     |One for Bob, one for me.
 |        |One for you, one for me.
 |Zaphod  |One for Zaphod, one for me.
+
```

Comment:
I wish GFM tables were a bit more flexible. What if we add a row (or replace the Zaphod row) with something like
```none
|(multiple lines) Carl \n Donna \n Eric|One for Eric, one for me.|
```